students/GKim/lesson02/series.py

```diff
@@ -0,0 +1,73 @@
+
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    a, b = 0, 1
+    for _ in range(n-1):
+        a, b = b, a + b
+    return b
+    
+
+
+def lucas(n):
+    """ compute the nth Lucas number """
+    a, b = 2, 1
+    if n == 0:
+        return a
+    elif n == 1:
```

Comment:
did you try without this check? turns out you don't need it ... can you see why?